Author: Giuseppe Iuculano <giuseppe@iuculano.it>
Description: Removed "p" from device name. A proper upgrade script is needed before using it.
--- 1.0.0.rc16-3/dmraid/lib/format/partition/dos.c
+++ 1.0.0.rc16-3/dmraid/lib/format/partition/dos.c
@@ -31,7 +31,7 @@ _name(struct lib_context *lc, struct rai
 {
 	const char *base = get_basename(lc, rd->di->path);
 
-	return type ? snprintf(str, len, "%s%s%u", base, OPT_STR_PARTCHAR(lc),
+	return type ? snprintf(str, len, "%s%u", base,
 			       partition) : snprintf(str, len, "%s", base);
 }
 
